R/bounds.r

```diff
@@ -1,3 +1,13 @@
+#' If needed, converts input from classes that lack division and multiplication
+#' to classes that can be scaled.
+#' @param x vector of values to convert, if needed
+adapt_input_class <- function(x) {
+  if (any(class(x) %in% c("Date", "POSIXct", "POSIXlt", "POSIXt"))) {
```

Comment:
Please use `inherits()` here.

A better formulation would be:

```R
if (...) {
  ...
} else {
 ...
}
```

as then you no longer need explicit `return()` statements..